### PR TITLE
Sass setup

### PR DESCRIPTION
**Pull request by ivanzusko**
1. Basic **SCSS** setup;
2. remove some unnecessary stuff;
3. `.scss` files created just to ensure that everything works properly. Will be refactored in parallel with applying UI/UX design

### CODE REVIEW

File: src/modules/Layout/Nav/Nav.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  &__item {
+    display: flex;
+    height: 100%;
+    align-items: center;
+    padding: 0 $hf-padding-middle;
+    color: $hf-color-white;
+    text-decoration: none;
+
+    &.active {
```
Comment (ivanzusko) [inline src/modules/Layout/Nav/Nav.scss]: `.active` is added by `react-router` **v4** , that's why in this particular case we don't use BEM--modificator stuff.

Comment (Vxplus) [inline src/modules/Layout/Nav/Nav.scss]: I think you can still use BEM for this as you can use the [activeClassName](https://github.com/reactjs/react-router-tutorial/tree/master/lessons/05-active-links#active-class-name)

Comment (ivanzusko) [inline src/modules/Layout/Nav/Nav.scss]: @Vxplus thank you!!! My bad. Fixed

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{
```
Comment (nik-kor) [inline .eslintrc]: I believe it makes sense to base on companies linting rules https://github.com/hellofresh/eslint-config-senf

File: package.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "axios": "^0.16.2",
     "history": "^4.6.3",
+    "node-sass-chokidar": "^0.0.3",
```
Comment (nik-kor) [inline package.json]: Why did you decide to pick up sass?

Comment (nik-kor) [inline package.json]: I'm asking cause in HF we mainly use PostCSS and CSS-in-JS.

File: src/modules/buttons/Button.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Button.css';
```
Comment (nik-kor) [inline src/modules/buttons/Button.jsx]: can the names be in lower case?

File: src/modules/buttons/Button.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+const propTypes = {
+  label: PropTypes.string,
+  type: PropTypes.string,
+};
+
+const Button = (props) => {
```
Comment (nik-kor) [inline src/modules/buttons/Button.jsx]: no need in curly braces

**Pull request merged**